api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/VisitTranslatorImplTest.java

```diff
@@ -70,7 +70,9 @@
 	
 	private static final String TYPE_CODE = "encounter-type-code";
 	
-	private static final String TYPE_DISPLAY = "encounter-type-display";
+	private static final String TYPE_DISPLAY = "Visit";
+	
+	private static final String VISIT_TYPE_CODE = "Visit";
```

Comment:
```suggestion
	private static final String VISIT_TYPE_CODE = "visit";
```

Shouldn't this be the value of `TYPE_CODE` above?